kafka-ui-api/src/main/java/com/provectus/kafka/ui/rest/MetricsRestController.java

```diff
@@ -0,0 +1,59 @@
+package com.provectus.kafka.ui.rest;
+
+import com.provectus.kafka.ui.api.ClustersApi;
+import com.provectus.kafka.ui.cluster.service.ClusterService;
+import com.provectus.kafka.ui.model.BrokerMetrics;
+import com.provectus.kafka.ui.model.Cluster;
+import com.provectus.kafka.ui.model.Topic;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ServerWebExchange;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/clusters")
+public class MetricsRestController implements ClustersApi {
```

Comment:
this should be generated from contract

Reply:
ClustersApi generated from contract, and I'm step-by-step changing mock endpoints to real ones from contract, is it ok?

Reply:
fixed, only contract used